docs/core/tools/dotnet-install-script.md

```diff
@@ -106,91 +96,45 @@ Specifies the URL for the Azure feed to the installer. It isn't recommended that
-- Two-part version in X.Y format representing a specific release (for example, `2.0` or `1.0`)
-- Branch name [for example, `release/2.0.0`, `release/2.0.0-preview2`, or `master` for the latest from the `master` branch ("bleeding edge" nightly releases)]
-
-The default value is `LTS`. For more information on .NET support channels, see the [.NET Core Support Lifecycle](https://www.microsoft.com/net/core/support) topic.
-
-`-Version <VERSION>`
-
-Represents a build version on the source channel (see the `-Channel` option). The possible values are:
-
-- `latest` - Latest build on the channel
-- `coherent` - Latest coherent build on the channel; uses the latest stable package combination
-- Three-part version in X.Y.Z format representing a specific build version (for example, `1.0.x` with `x` representing the patch version; or a specific build, such as `2.0.0-preview2-006120`)
-
-If omitted, `-Version` defaults to the first [global.json](global-json.md) that contains the `version` member. If that isn't present, `-Version` defaults to `latest`.
-
-`--install-dir <DIRECTORY>`
-
-Specifies the installation path. The directory is created if it doesn't exist. The default value is `$HOME/.dotnet`.
-
-`--architecture <ARCHITECTURE>`
-
-Architecture of the .NET Core binaries to install. Possible values are `auto`, `x64` and `amd64`. The default value is `auto`, which represents the currently running OS architecture.
-
-`--shared-runtime`
-
-If set, this switch limits installation to the shared runtime. The entire SDK isn't installed.
-
-`--debug-symbols`
-
-If set, the installer includes debugging symbols in the installation.
-
-> [!NOTE]
-> This switch is not currently avaiable but planned for a future release.
-
-`--dry-run`
-
-If set, the script won't perform the installation; but instead, it displays what command line to use to consistently install the currently requested version of the .NET Core CLI. For example if you specify version `latest`, it displays a link with the specific version so that this command can be used deterministically in a build script. It also displays the binary's location if you prefer to install or download it yourself.
-
-`--no-path`
-
-If set, the prefix/installdir are not exported to the path for the current session. By default, the script will modify the PATH, which makes the CLI tools available immediately after install.
-
 `--verbose`
 
 Display diagnostics information.
 
-`--azure-feed`
-
-Specifies the URL for the Azure feed to the installer. It isn't recommended that you change this value. The default is `https://dotnetcli.azureedge.net/dotnet`.
-
 `--help`
 
 Prints out help for the script.
 
 ## Examples
 
-Install the latest development version to the default location:
+Install the latest long-term supported version to the default location:
+
+Windows:
+
+`./dotnet-install.ps1 -Channel LTS`
+
+macOS/Linux:
+
+`./dotnet-install.sh --channel LTS`
+
+Install the latest 2.0 version to the specified location:
```

Comment:
latest 2.0? this will get outdated fast. Install the .NET Core 2.0 SDK to the specified location perhaps?

Reply:
We are not necessarily installing the .NET Core 2.0 SDK; the command is installing the latest version from 2.0 channel - updated as such...

Reply:
Better!